modules/__tests__/toInclude-test.js

```diff
@@ -56,6 +60,52 @@ describe('toInclude', () => {
     }).toThrow(/to include/)
   })
 
+  it('compares partial object only when both expected and actual properties are object', () => {
+    expect(() => {
+      expect({ a: { b: 2 } }).toInclude({ a: {} })
+    }).toNotThrow()
+
+    expect(() => {
+      expect({ a: 1 }).toInclude({ a: {} })
+    }).toThrow(/to include/)
+
+    expect(() => {
+      expect({ a: [] }).toInclude({ a: {} })
+    }).toThrow(/to include/)
+
+    expect(() => {
+      expect({ a: '1' }).toInclude({ a: {} })
+    }).toThrow(/to include/)
+
+    expect(() => {
+      expect({ a: () => {} }).toInclude({ a: {} })
+    }).toThrow(/to include/)
+  })
+
+  if (typeof Object.create === 'function') {
+    it('ignores nonenumerable properties of an expected object', () => {
+      expect(() => {
+        expect({}).toInclude(Object.create({}, { a: { value: 1 } }))
+      }).toNotThrow()
+    })
+  }
+
+  if (typeof Symbol === 'function') {
+    const symbol = Symbol()
+
+    it('does not throw when an object contains an expected object with a symbol key', () => {
+      expect(() => {
+        expect({ [symbol]: 1, b: 2 }).toInclude({ [symbol]: 1 })
```

Comment:
Symbols are supported in plenty of environments where computed property keys are not - could you construct the objects the old-fashioned way?